package.json

```diff
@@ -24,7 +24,7 @@
     "test-server": "babel-tape-runner test/server.jsx | tap-spec",
     "tdd-server": "nodemon -x npm -i node_modules/ -e js,jsx -- run -s test-server",
     "tdd-browser": "mkdir -p .cache; mkdir -p .cache/test; touch .cache/test/browser.js 2> /dev/null; watchify test/browser.jsx --transform babelify --external react/addons --external react/lib/ReactContext --external react/lib/ExecutionEnvironment --debug --outfile .cache/test/browser.js & devtool --watch .cache/test/browser.js --console --browser-field --no-node-timers --show .cache/test/browser.js | faucet",
-    "watch": "babel index.js --watch --out-file index.es5",
```

Comment:
This won't work because it'll overwrite the original file!

We should probably just move to a `dist` and `src` folder pattern that's become common.